### PR TITLE
Alexandria 18 setup react front end service

### PR DESCRIPTION
**Pull request by JessHua159**
The `book-exchange-website` folder contains the initial React.js setup. You can follow steps 8 and 9 to see a local build of the website. I am not sure if the `src` folder with the initial Spring Boot setup is supposed to be separate from or be in the folder with the React.js setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,5 +24,11 @@ spring:
 ```
 
 7. Set application run configuration to `local` using the following flag: `spring.profiles.active=local`. 
+8. Install a recent version of Node.js from [here](https://nodejs.org/en/download/).
+9. Front-end local build: 
+```
+cd book-exchange-website
```
Comment (anirban-a) [inline README.md]: please rename this to `alexandria-ui` instead of `book-exchange-website`

File: book-exchange-website/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "book-exchange-website",
```
Comment (anirban-a) [inline book-exchange-website/package.json]: please rename this as mentioned above.

**Review by anirban-a — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JessHua159) [reply README.md]: I renamed it to `alexandria-ui` as another commit.

File: book-exchange-website/package.json (hunk 2 above)
Comment (JessHua159) [reply book-exchange-website/package.json]: I renamed it to `alexandria-ui` as another commit.

**Review by anirban-a — APPROVED** [review]

**Review by MMosley502 — APPROVED** [review]

**Pull request closed without merging**